.node-version

```diff
@@ -0,0 +1 @@
+20.8.0
```

Comment:
I thought we need it so I added

Reply:
Nice different version ?

https://github.com/tabi-memo/frontend/blob/95cbd38b1b914c92e66f4b5e154fea84c70afaf3/.github/workflows/lint.yml#L17

I'd say both have to be same version.

Reply:
@Yo-mah-Ya 
Thank you! I changed it 
https://github.com/tabi-memo/frontend/pull/9/commits/f7e13af6c086c73b7f80367263cde6c11d84ffe6